monix-reactive/shared/src/main/scala/monix/reactive/internal/operators/OnErrorRetryWithBackoffObservable.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2014-2020 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.reactive.internal.operators
+
+import monix.execution.Ack.Continue
+import monix.execution.FutureUtils.timeout
+import monix.execution.cancelables.OrderedCancelable
+import monix.execution.{Ack, Cancelable, Scheduler}
+import monix.reactive.Observable
+import monix.reactive.observers.Subscriber
+
+import scala.concurrent.Future
+import scala.concurrent.duration.FiniteDuration
+
+private[reactive] final class OnErrorRetryWithBackoffObservable[+A](
+                                                                     source: Observable[A],
```

Comment:
The formatting seems off, it should be on the left - how did you run `scalafmt`?

Reply:
I ran scalafmt as instructed in the CONTRIBUTING.md, I am a little concerned that it might have used IntelliJ's scalafmt config and not the projects. I will fix this.